cli/cache/cache_db.rs

```diff
@@ -42,7 +42,7 @@ impl CacheDBConfiguration {
   fn create_combined_sql(&self) -> String {
     format!(
       "
-      PRAGMA journal_mode=TRUNCATE;
+      PRAGMA journal_mode=WAL;
```

Comment:
It seems we previously disabled this for performance reasons? https://github.com/denoland/deno/pull/18084

I wonder what impact this will have on non-gcp machines (not sure why that machine is so slow because neither my winows, mac, or linux machine have this issue)

Reply:
Oh, just realized that was done only for startup perf reasons, but we initialize the sqlite caches on blocking threads now.

Reply:
I think that was in https://github.com/denoland/deno/pull/18401, and we benchmarked almost all of them within a 10th of a ms.

Reply:
I just benchmarked WAL and it is much faster on my mac. 176ms down to 142ms.